gulpfile.babel.js

```diff
@@ -103,5 +144,9 @@ gulp.task('watch', function() {
 })
 
 gulp.task('default', gulp.series(
-  'rebuild', 'spec', 'watch'
+  'config', 'rebuild', 'spec', 'watch'
 ))
+
+gulp.task('reset', gulp.series(
+  'config:reset', 'default'
+))
```

Comment:
Could you add a trailing newline here? GitHub marks it with the lil' red arrow sign if it's missing.